modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/BlockStreamTest.kt

```diff
@@ -51,18 +50,13 @@ class BlockStreamTest : IrohaTest<Iroha2Client>(account = ALICE_ACCOUNT_ID, keyP
         var blocks = mutableListOf<VersionedBlockMessage>()
         blocksResult.collect { block -> blocks.add(block) }
 
-        var instructions = checkBlockStructure(
-            blocks[0],
-            1,
-            GENESIS,
-            GENESIS,
-            4
```

Comment:
@gv-timur the number of instructions in genesis can change, as actually happened